### PR TITLE
add unit test for selector

### PR DESCRIPTION
**Pull request by lixingchenDaoCloud**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

add unit test for selector

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

NONE

### CODE REVIEW

File: pkg/utils/expression/selector_test.go [hunk 1]
```diff
@@ -0,0 +1,150 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```
Comment (wzshiming) [inline pkg/utils/expression/selector_test.go:2]:
```suggestion
Copyright 2023 The Kubernetes Authors.
```

Comment (lixingchenDaoCloud) [reply pkg/utils/expression/selector_test.go]: Thanks for your review ,and i have change this problem

**Review by wzshiming — APPROVED** [review]
/approve
/lgtm

**Pull request merged**